src/index.js

```diff
@@ -148,14 +149,13 @@ async function bootstrap(state) {
 
 async function moveToProduction() {
   log.info('🚚 starting move to production');
-  await client.copyIndex(c.indexName, c.bootstrapIndexName, [
```

Comment:
we no longer copy the settings from the prod index to the bootstrap index, but rely on the bootstrap index being correct